Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -1,7 +1,144 @@
 ﻿using System;
-
+using System.Text.RegularExpressions;
 namespace Tavisca.Bootcamp.LanguageBasics.Exercise1
 {
+    class Equation{
+        private string operand1;
+        private string expectedOperand1;

```

Comment:
you don't need to store expectedOperand1/ expectedOperand2/expectedResult here, because what is expected and what is given is conditional. Your classes should contain members which are required for on object to function.  expected values needs to be derived, they should not be class members here